### PR TITLE
Chore/updating tooling and style

### PR DESCRIPTION
**Pull request by icazevedo**
#### What problem is this solving?

This PR updates the tooling used by this project. With it, it updates the code style and adds the ability to use custom classes on the CSS Handles.

#### How to test it?
Look at the "What's trending shelf". I've adjusted the threshold on this workspace so the product-availability block is visible.

[Workspace](https://icarovtex--b2bstoreqa.myvtex.com/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Updated tooling and code style
+
```
Comment (igorbrasileiro) [inline CHANGELOG.md]: It doesn't be part of the changelog because don't be part of this app's features.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "*.{json,graphql,gql}": [
+      "prettier --write"
+    ]
+  },
+  "devDependencies": {
+    "@vtex/prettier-config": "^0.1.4",
+    "eslint": "^7.12.1",
+    "eslint-config-vtex": "^12.3.2",
+    "eslint-config-vtex-react": "^6.3.2",
+    "husky": "^4.2.3",
```
Comment (igorbrasileiro) [inline package.json]: Can use the new husky@5.x.

**Review by igorbrasileiro — APPROVED** [review]
LGTM, Just fix the CHANGELOG

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "product-availability",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "lint": "eslint --ext js,jsx,ts,tsx .",
+    "test": "cd ./react && yarn test",
+    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\""
+  },
+  "husky": {
```
Comment (igorbrasileiro) [inline package.json]: This husky setting doesn't need anymore, you should run this command `npx husky add .husky/pre-commit "yarn lint-staged"`. Take a look here https://typicode.github.io/husky/#/?id=add-a-hook

File: react/components/ProductAvailability.tsx [hunk 4]
```diff
@@ -1,48 +1,53 @@
...
-  if (availableQuantity == null || availableQuantity < 1) {
+function ProductAvailability({
+  threshold,
+  lowStockMessage,
+  highStockMessage,
+  availableQuantity,
+}: Props) {
+  if (
+    availableQuantity == null ||
+    availableQuantity === undefined ||
```
Comment (victorhmp) [inline react/components/ProductAvailability.tsx]: `availableQuantity == null` will be `false` if `availableQuantity` is `undefined`, so this check is redundant.

You could either remove it or change the first one to `availableQuantity === null`. Ooor, you could even do `!Boolean(availableQuantity)`

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: react/components/ProductAvailability.tsx (hunk 4 above)
Comment (icazevedo) [reply react/components/ProductAvailability.tsx]: The lint was angry about `!Boolean(availableQuantity)` so I just `!availableQuantity` 😄

**Review by victorhmp — APPROVED** [review]

**Pull request merged**